src/components/Button.js

```diff
@@ -0,0 +1,30 @@
+import {StyleSheet, Text, TouchableOpacity} from 'react-native';
+import React from 'react';
+
+const Button = ({text, color, onHandlePress}) => {
```

Comment:
We should opt for better name for this component, as `react-native` already have a `Button` component.

Reply:
mark it as resolved